content/docs/run-core-node/installation.mdx

```diff
@@ -7,15 +7,34 @@ There are three ways to install Stellar Core: you can use a Docker image, use pr
 
 ## Docker-based installation
 
-SDF maintains a [quickstart image](https://github.com/stellar/docker-stellar-core-horizon) that bundles Stellar Core with Horizon and postgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers.
+### Development environments
 
-The SDF also maintains a Stellar-Core-only [standalone image](https://github.com/stellar/docker-stellar-core) that starts a 3 node local stellar-core network, all on the same docker host
+SDF maintains a [quickstart image](https://github.com/stellar/docker-stellar-core-horizon) that bundles Stellar Core with Horizon and postgreSQL databases. It's a quick way to set up a default, non-validating, ephemeral configuration that should work for most developers.
 
 In addition to SDF images, Satoshipay maintains separate Docker images for [Stellar Core](https://github.com/satoshipay/docker-stellar-core) and [Horizon](https://github.com/satoshipay/docker-stellar-horizon). The Satoshipay Stellar Core Docker image comes in a few flavors, including one with the AWS CLI installed and one with the Google Cloud SDK installed. The Horizon image supports all Horizon environment variables.
 
+### Production environments
+
+The SDF also maintains a Stellar-Core-only [standalone image](https://hub.docker.com/repository/docker/stellar/stellar-core). The image utilizes deb packages so the stellar-core binary checksum can be verified.
+
+Example usage:
+
+```
+docker run stellar/stellar-core:latest help
+docker run stellar/stellar-core:latest gen-seed
+```
+
+To run daemon you need to provide a configuration file:
```

Comment:
Link to the [example configuration file](https://github.com/stellar/stellar-core/blob/master/docs/stellar-core_example.cfg) here?

Reply:
Good idea. I'll need to think about it a bit more because simple in-container sqlite DB from the example you linked to won't work out of the box. But it's certainly a good idea to have something that works.